.github/workflows/terraform-ci.yml

```diff
@@ -0,0 +1,67 @@
+name: 'Terraform-ci'
+
+on: [push, pull_request]
+
+defaults:
+  run:
+    shell: sh
+
+jobs:
+  # Terraform init and validate configuration
+  terraform-init:
+    name: 'Terraform-init'
+    runs-on: ubuntu-latest
+    environment: dev
+    container:
+      image: hashicorp/terraform:0.14.6
```

Comment:
maybe it makes sense to always use the latest version of terraforms for checks?

Reply:
Please describe distinct Issue for it

Reply:
Created draft issue, please check it
https://github.com/maddevsio/aws-eks-base/issues/35